pyClient/commands/zeth_ls_notes.py

```diff
@@ -4,13 +4,16 @@
 
 from commands.utils import open_web3_from_ctx, load_zeth_address_secret, \
     open_wallet, load_mixer_description_from_ctx
-from click import command, pass_context
+from zeth.utils import EtherValue
+from click import command, option, pass_context
 from typing import Any
 
 
 @command()
+@option("--balance", is_flag=True, help="Show total balance")
+@option("--spent", is_flag=True, help="Show spent notes")
 @pass_context
-def ls_notes(ctx: Any) -> None:
+def ls_notes(ctx: Any, balance: bool, spent: bool) -> None:
```

Comment:
Same here, `Any` should be `Context`

Reply:
(fixed)

Reply:
Thanks @dtebbs. I like the changes you have made to display the balance and the spent notes!